.github/workflows/ci.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
     strategy:
       matrix:
-        go-version: [1.19.x, 1.20.x]
+        go-version: [1.19.x, 1.21.x, 1.22.x]
```

Comment:
We can drop 1.19

Reply:
Yeah, my thinking here was to keep a minimum version (unless we had a strong reason to drop it), then check current versions.

I'm fine with dropping 1.19 by now probably, although "if it works", then 🤷 .

I'll drop the 1.20, and keep the upper-bound to 1.21, 1.22 as suggested.

Reply:
oh! I read the diff wrong, thought it was a suggestion, but I already changed to 1.21, 1.22 🤦 😂